stdlib/public/SDK/Foundation/NSStringAPI.swift

```diff
@@ -1023,20 +1027,19 @@ extension String {
     scheme tagScheme: String,
     options opts: NSLinguisticTaggerOptions = [],
     orthography: NSOrthography? = nil,
-    tokenRanges: UnsafeMutablePointer<[Range<Index>]> = nil // FIXME:Can this be nil?
+    tokenRanges: UnsafeMutablePointer<[Range<Index>]>? = nil // FIXME:Can this be nil?
   ) -> [String] {
     var nsTokenRanges: NSArray? = nil
-    let result = tokenRanges._withBridgeObject(&nsTokenRanges) {
+    let op = {
```

Comment:
I argue that it is a bad idiom, and that two tiny methods, each of which is only used twice, is not enough weight to be worth the encapsulation. Both instances of the related `_withBridgeValue` is easier to understand now; the two instances of `_withBridgeObject` are not quite as clear-cut but still seem like improvements to me. (At the very least my renamed helper function makes a lot more sense.)